Library/Homebrew/formula.rb

```diff
@@ -1515,6 +1515,18 @@ def self.installed
     end.uniq(&:name)
   end
 
+  # An array of installed {Formula} that are dependencies of other installed {Formula}
+  # @private
+  def self.installed_deps(formulae = installed)
+    formulae.flat_map(&:runtime_formula_dependencies).uniq(&:name)
+  end
```

Comment:
```suggestion
  # An array of installed {Formula} that are dependencies of other installed {Formula}
  # @private
  def self.installed_deps(formulae = installed)
    formulae.flat_map(&:runtime_formula_dependencies).uniq(&:name)
  end
```

Let's avoid adding this new function that isn't called anywhere.

Reply:
Removed